pkg/volume/util/util.go

```diff
@@ -614,6 +616,22 @@ func FsUserFrom(pod *v1.Pod) *int64 {
 	return fsUser
 }
 
+func HonorFsUser(f *FileProjection, fsUser *int64) {
+	// We are adding this function out of extra cautious, to not do this if
+	// the feature flag is not active.
```

Comment:
```suggestion
	// This function provides some defence-in-depth so that looking up an intended
	// owner user only applies when the UserNamespacesSupport feature gate
	// is enabled.
```

Reply:
I think defense-in-depth is a loaded word, typically used for a different thing. It seems confusing to use it in this context, where it is not any of the common meanings.

This is not about "_looking up_ owner user" either.

I don't think this improves the comment. Please re-open if you think something is not clear and explain what isn't clear.

Reply:
I'm fine with a different comment from what I suggested. However:
- “we” is unclear (unless you fire up `git annotate`)
- try to write a [timeless](https://developers.google.com/style/timeless-documentation) comment. The text in the PR is too much about the initial context and may go stale.

I've guessed at what the intention was here to trigger discussion. Another reviewer can spot that I was confused here and might be able to decide if extra clarification is useful.

Reply:
@sftim nice link about timless comments, thanks.

With "we" I meant the Kubernetes authors, nothing to look up in git history. What is not timeless about this comment, though?